### ISSUE TITLE
Add other libraries to nuget

### ISSUE THREAD

**Issue by annymsMthd** [raspberry-sharp/raspberry-sharp-io#58]
Doing some work on the pi and it would be useful to have the other libraries in this repo on nuget.

**Comment by BennyM** [comment]
Indeed, I was looking for Raspberry.IO.Components today.

**Comment by xlevel** [comment]
I agree, having Raspberry.IO.Components available via Nuget would be really great.

**Comment by Petermarcu** [comment]
I just hit the same thing. @raspberry-sharp is that something you want to do? I can try to make a PR that would start creating the packages.
